pkg/pipelines/tekton/tasks.go

```diff
@@ -1,5 +1,14 @@
----
-apiVersion: tekton.dev/v1
+package tekton
+
+import (
+	"fmt"
+	"strings"
+)
+
+var DeployerImage = "ghcr.io/knative/func-utils:latest"
```

Comment:
This variable would be set via linker flags in productized version.

@dsimansk where do we set the linker flags for `kn`?

Reply:
The soonest midstream, the latest possible is internal build. I'll follow-up on chat.

Reply:
Let's add a placeholder ldflag to `Makefile`. Nothing fancy, just current default value. For a demonstration and "contract definition" purposes.